.circleci/config.yml

```diff
@@ -111,6 +127,10 @@ workflows:
               only: /.*/
           context:
             - auth0-dev
+      - pre-release-test:
+          filters:
+            tags:
+              only: /.*/
```

Comment:
youre prob tracking but rn this will run on all commits

Reply:
yup will restrict to main branch

Reply:
do we want it to run pre publishing snapshots or just releases

Reply:
just releases

Reply:
there's an if case that terminates early if there is no git tag